ApiBundle/EventSubscriber/AbstractSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\EventSubscriber;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\KernelEvent;
+
+/**
+ * Class AbstractSubscriber
+ */
+abstract class AbstractSubscriber
+{
+    protected $resolver;
+    protected $annotationReader;
+
+    /**
+     * @param AnnotationReader            $annotationReader
+     * @param ControllerResolverInterface $resolver
+     */
+    public function __construct(AnnotationReader $annotationReader, ControllerResolverInterface $resolver)
+    {
+        $this->resolver = $resolver;
+        $this->annotationReader = $annotationReader;
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return boolean
+     */
+    protected function isApiRequest(Request $request)
+    {
+        return 0 === strpos($request->get('_route'), 'open_orchestra_api');
```

Comment:
I had this one : https://trello.com/c/Z2NZGPOL/883-code-on-check-par-endroit-si-la-requete-est-une-requete-d-api-en-testant-que-le-prefixe-de-la-route-est-open-orchestra-api-ce-qu